__tests__/app.test.js

```diff
@@ -176,4 +176,76 @@ describe('GET: /api/topics', () => {
             expect(msg).toBe("Invalid input");
           });
       });
-    });
\ No newline at end of file
+    });
+
+    describe('POST /api/articles/:article_id/comments', () => {
+      test('status 201: Posts comment to corresponding article and responds with comment', () => {
+        const testComment = {
+          username: 'butter_bridge',
+          body: 'Hello!',
+        };
+        return request(app)
+          .post('/api/articles/1/comments')
+          .send(testComment)
+          .expect(201)
+          .then(({ body }) => {
+            const { comment } = body;
+            expect(comment).toMatchObject({
+              comment_id: 19,
+              votes: 0,
+              created_at: expect.any(String),
+              author: 'butter_bridge',
+              body: 'Hello!',
+              article_id: 1,
+            });
+          });
+      })
+    test("status: 400, responds with error message when request is sent with empty object", () => {
+      const testComment = {}
```

Comment:
can we add another test for:
Status 400, invalid ID
Status 201 where  it ignores unnecessary properties eg adding another value into the object you pass in